### PR TITLE
feat: Add HIGH resource for cpu and ram

### PR DESCRIPTION
**Pull request by s-yoshika**
We mainly use `k8s-executor` plugin in our cluster, and some builds fail due to lack of resources.
So, This PR add `HIGH` resource for cpu and ram to `k8s` executor just like [k8s-vm](https://github.com/screwdriver-cd/executor-k8s-vm/pull/10).

### References
[758](https://github.com/screwdriver-cd/screwdriver/issues/758), [759](https://github.com/screwdriver-cd/screwdriver/issues/759)

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -215,34 +217,71 @@ describe('index', function () {
...
+                container: testContainer,
+                token: testToken,
+                apiUri: testApiUri
+            }).then(() => {
+                assert.calledOnce(requestMock);
+                assert.calledWith(requestMock, postConfig);
+            });
+        });
+
+        it('sets the memory appropriately when cpu is set to HIGH', () => {
```
Comment (tkyi) [inline test/index.test.js]: sets the `cpu` appropriately

File: index.js [hunk 2]
```diff
@@ -56,6 +60,10 @@ class K8sExecutor extends Executor {
      * @return {Promise}
      */
     _start(config) {
+        const cpuConfig = hoek.reach(config, 'annotations', { default: {} })[CPU_RESOURCE];
+        const ramConfig = hoek.reach(config, 'annotations', { default: {} })[RAM_RESOURCE];
+        const CPU = (cpuConfig === 'HIGH') ? 6 : 2;
```
Comment (minz1027) [inline index.js]: CPU config is kinda different for kubernetes, can you double check?
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/

Comment (s-yoshika) [reply index.js]: The integer format of CPU config has been deprecated in `Kubernetes v1.8`.
So, `millicpu` format seems to be prefered.I'll fix it. Thanks!

File: test/index.test.js (hunk 1 above)
Comment (s-yoshika) [reply test/index.test.js]: Sorry, I'll fix it!

**Pull request merged**